config/dictionaries/resource.json

```diff
@@ -543,8 +543,7 @@
         "summary": "Update a device tag",
         "description": "",
         "method": "PATCH",
-        "endpoint": "/v6/device_tag",
-        "filters": "?\\$filter=(tag_key%20eq%20'<EXISTING-KEY>')%20and%20(device%20eq%20<DEVICE_ID>)",
+        "endpoint": "/v6/device_tag(<TAG-ID>)",
```

Comment:
Optional-idea: As an additional/alternative suggestion, it might be more convenient to suggest using the composite ID notation, which is also save against unescaped `$filter`s.
```suggestion
        "endpoint": "/v6/device_tag(device=<DEVICE_ID>,tag_key='<EXISTING-KEY>'",
```

If you prefer having the numeric ID variant as well, but also like my suggestion you can add it as a separate example by coping the current one and using a different id and summary.
Eg: `"id": "update-tag-by-natural-key", "summary": "Update a device tag by device id and tag_key"`